src/genie/libs/parser/iosxe/show_lisp.py

```diff
@@ -1582,11 +1585,15 @@ def cli(self, service, instance_id, output=None):
 
         #   Locator  Uptime    State      Pri/Wgt     Encap-IID
         #   10.1.8.8 00:04:02  up          50/50        -
+        # State can be: up, down, route-rejec plus self
         p5 = re.compile(r'(?P<locator>(\S+)) +(?P<uptime>(\S+))'
-                         ' +(?P<state>(up|down))'
+                         ' +(?P<state>(up|down|route-rejec)) *,? *(?P<self>(self)?)'
                          ' +(?P<priority>(\d+))\/(?P<weight>(\d+))'
                          '(?: +(?P<encap_iid>(\S+)))?$')
 
+        # Encapsulating to proxy <device>
```

Comment:
better to add actual sample output instead of place holder. so that we can verify if the regex works agains the sample output.
```suggestion
        # Encapsulating to proxy ETR
        # Encapsulating to proxy ETR Encap-IID 3
```